### PR TITLE
Update Dockerfile for label data dir

### PR DESCRIPTION
**Pull request by pdmack**
Make sure we keep the label, column data dir for the existing pipelines.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -146,6 +146,7 @@ COPY "./docker" "./docker"
 COPY "./docs" "./docs"
 COPY "./examples" "./examples"
 COPY "./models" "./models"
+COPY "./morpheus/data" "./morpheus/data"
```
Comment (mdemoret-nv) [inline docker/Dockerfile:149]: This directory gets installed now with the `morpheus` python package. If you run:

```bash
$ python -c "import morpheus;print(morpheus.DATA_DIR)"
$HOME/Repos/morpheus/morpheus-dev/morpheus/data
```
Is this path used by scripts anywhere?

**Review by mdemoret-nv — COMMENTED** [review]
The data path is now installed as part of the python/conda package. Does this PR need to be more about documentaiton/scripting updates?

File: docker/Dockerfile (hunk 1 above)
Comment (pdmack) [reply docker/Dockerfile:149]: Let me double check things once we have a working release build. I'll validate that our external instructions are correct.

Comment (mdemoret-nv) [reply docker/Dockerfile:149]: Release build should be working as of #164

**Comment by mdemoret-nv** [comment]
Closing this in favor of #232.

**Pull request closed without merging**